distributed/scheduler.py

```diff
@@ -940,6 +942,7 @@ def __init__(
             'heartbeat_worker': self.heartbeat_worker,
             'get_task_status': self.get_task_status,
             'get_task_stream': self.get_task_stream,
+            'add_preload_function': self.register_preload_function
```

Comment:
I think that whatever is chosen should be named the same on the client and the scheduler - I don't see any benefit to coming up with two different names for this?

For me I associate the functionality with *initializing* a worker whereas `preload` seems a bit more ambiguous - `pre-` what? Given that I think `register_init_func` is a better name.

Reply:
Thanks for the comment, I did use two different names just to be sure I understood what was happening behind 🙂, just changed that and adopted your suggestion.